tests/components/rfxtrx/test_binary_sensor.py

```diff
@@ -0,0 +1,208 @@
+"""The tests for the Rfxtrx sensor platform."""
+from datetime import timedelta
+
+from homeassistant.setup import async_setup_component
+from homeassistant.util.dt import utcnow
+
+from . import _signal_event
+
+from tests.common import async_fire_time_changed
+
+
+async def test_default_config(hass, rfxtrx):
+    """Test with 0 sensor."""
+    await async_setup_component(
+        hass, "binary_sensor", {"binary_sensor": {"platform": "rfxtrx", "devices": {}}}
+    )
+    await hass.async_block_till_done()
+
+    assert len(hass.states.async_all()) == 0
+
+
+async def test_one(hass, rfxtrx):
+    """Test with 1 sensor."""
+    await async_setup_component(
+        hass,
+        "binary_sensor",
+        {
+            "binary_sensor": {
+                "platform": "rfxtrx",
+                "devices": {"0a52080705020095220269": {"name": "Test"}},
+            }
+        },
+    )
+    await hass.async_block_till_done()
+
+    state = hass.states.get("binary_sensor.test")
+    assert state
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5802827)